### DISCUSSION TITLE
Score page, widget for all habits

### DISCUSSION THREAD

**Discussion by hosnifraj1** [iSoron/uhabits#1818] [Feature Requests]
Sometimes, when I want to check all my habits score, I have to go through _every single habit_ ( frustrating for people who has tons of active habits ). Also, a lot of habits tracker apps has this useful feature which let the user do a quick score check for all habits and see what is the most active habit. I hope developers will understand and put this on their road map as I believe it will benefit a lot of people.

**Comment by ghost** [comment]
Agree!  Need a score summary page for all habits.  By month and year.

**Comment by morganwaser** [comment]
Would love this feature. I have set some long term goals for the year but to achieve them I am working with smaller weekly and monthly goals to stay on track and build my way their. Having a Sunday of everything would be awesome.
